styx-scheduler-service/src/main/java/com/spotify/styx/api/TriggerRequest.java

```diff
@@ -0,0 +1,51 @@
+/*-
+ * -\-\-
+ * Spotify Styx Scheduler Service
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.spotify.styx.model.TriggerParameters;
+import com.spotify.styx.model.WorkflowId;
+import io.norberg.automatter.AutoMatter;
+import java.util.Optional;
+
+@AutoMatter
+public interface TriggerRequest {
+
+  @JsonProperty("workflow_id")
+  WorkflowId workflowId();
+
+  @JsonProperty("parameter")
+  String parameter();
+
+  @JsonProperty("parameters")
```

Comment:
Now we have `"parameter"` and `"parameters"`... thinking of maybe going for `"trigger_parameters"` here, or collapsing the above two into `"workflow_instance_id"`.

Reply:
Renaming to `triggerParameters` LGTM. Also we align with that in `BackfillInput`.